src/packagedef.jl

```diff
@@ -98,9 +99,17 @@ function startdebug(socket, error_handler = nothing)
                     ret = if msg.cmd == :continue
                         our_debug_command(:c, state)
                     elseif msg.cmd == :next
-                        our_debug_command(:n, state)
+                        our_debug_command(:nc, state)
```

Comment:
That is not what we want, right? I think this is the "step over" case, so we want `:n` here?

Reply:
Yes(-ish).
The DAP docs say
```
interface NextArguments {
  /**
   * Execute 'next' for this thread.
   */
  threadId: number;

  /**
   * Optional granularity to step. If no granularity is specified, a granularity of 'statement' is assumed.
   */
  granularity?: SteppingGranularity;
}
```
so in theory we should only do statement steps here.

"Line" steps make more sense, and I forgot to revert this before pushing.

Reply:
Anyhow, revert this change.